### PR TITLE
Spark Operator 2.4.4-0.2.0 release

### PR DESCRIPTION
**Pull request by alembiewski**
This PR contains the release of Spark Operator 2.4.4-0.2.0.

Changelist:
* Standardize Operator version (semver compatible)
* Spark Operator Docker image based on [apache/spark](https://github.com/apache/spark/) 2.4.4 with Hadoop 2.9.2 support
* Added Python and R support to Spark Operator image
* Added support for automatic installation of monitoring resources
* Added configuration parameters and documentation for HA installation
* Added documentation describing integration with Volcano batch scheduler

Tests are available here: https://github.com/mesosphere/kudo-spark-operator/pull/87

### CODE REVIEW

File: repository/spark/docs/2.4.4-0.2.0/installation.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        │   └── Step rbac [COMPLETE]
+        ├── Phase webhook (serial strategy) [COMPLETE]
+        │   └── Step webhook [COMPLETE]
+        ├── Phase spark (serial strategy) [COMPLETE]
+        │   └── Step spark [COMPLETE]
+        ├── Phase monitoring (serial strategy) [COMPLETE]
+        │   └── Step monitoring [COMPLETE]
+        └── Phase history (serial strategy) [COMPLETE]
+            ├── Step history-deploy [COMPLETE]
+            └── Step history-service [COMPLETE]
```
Comment (akirillov) [inline repository/spark/docs/2.4.4-0.2.0/installation.md:40]: is this output still up to date?

File: repository/spark/docs/2.4.4-0.2.0/installation.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spark-operator-2
+  labels:
+    name: spark-operator-2
+EOF
+```
+
+
```
Comment (akirillov) [inline repository/spark/docs/2.4.4-0.2.0/installation.md]: extra-newline

File: repository/spark/docs/2.4.4-0.2.0/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Release Notes
+---
+
+## 2.4.4-0.2.0 (latest)
+* Spark Operator Docker image based on [apache/spark](https://github.com/apache/spark/) 2.4.4 with Hadoop 2.9.2 support
+* Added Python and R support to Spark Operator image
+* Added support for automatic installation of monitoring resources
+* Added configuration parameters and documentation for HA installation
+* Added documentation describing integration with Volcano batch scheduler
```
Comment (akirillov) [inline repository/spark/docs/2.4.4-0.2.0/release-notes.md:9]: 👍

**Review by akirillov — APPROVED** [review]
Thanks, @alembiewski! LGTM 👍

File: repository/spark/docs/2.4.4-0.2.0/installation.md (hunk 1 above)
Comment (alembiewski) [reply repository/spark/docs/2.4.4-0.2.0/installation.md:40]: yes

File: repository/spark/docs/2.4.4-0.2.0/installation.md (hunk 2 above)
Comment (alembiewski) [reply repository/spark/docs/2.4.4-0.2.0/installation.md]: fixed

**Pull request merged**